lib/render.rb

```diff
@@ -3,6 +3,7 @@
 class Render
 
   def render(*boards, reveal)
+    # TO DO: ^ tests are very unhappy -- probably have to put reveal first
```

Comment:
I don't know if we can put it first... But ultimately it shouldn't matter, we just can't leave it out anymore.